src/config/LaunchSettings.cpp

```diff
@@ -253,7 +253,6 @@ bool LaunchSettings::ExtractorTool(std::wstring_view wud_path, std::string_view
 		catch (const std::exception& ex)
 		{
 			cemuLog_log(LogType::Force, "can't write file: {}", ex.what());
-			puts(fmt::format("can't write file: %s\n", ex.what()).c_str());
```

Comment:
Are you sure this line isn't used? If I had to guess puts might be used here to output things to the command line. I'd probably just leave it in.

Reply:
I wasn't sure but it showed up in your recommended changes so I assumed that it wasn't required.
I'll stick it back in.